worlds/blasphemous/__init__.py

```diff
@@ -258,18 +260,24 @@ def pre_fill(self):
 
         if self.multiworld.thorn_shuffle[self.player] == 1:
             self.multiworld.local_items[self.player].value.add("Thorn Upgrade")
+
+
+    def get_pre_fill_items(self) -> List["Item"]:
+        return self.pre_fill_items
         
 
     def place_items_from_set(self, location_set: Set[str], name: str):
         for loc in location_set:
             self.multiworld.get_location(loc, self.player)\
                 .place_locked_item(self.create_item(name))
+            self.pre_fill_items.append(self.create_item(name))
```

Comment:
It looks like you're adding these to the list after you actually place them? If you don't need to keep a list of the pre_fill items on your world don't bother with it; it's just a recommendation. The important part is that you don't submit items to the multiworld itempool and then take them back out.

Reply:
Yeah I'll admit that I kind of just skimmed the discord thread and assumed that `get_pre_fill_items` was used during some other step of generation, so I put it in. I'll just get rid of it now then